src/scripts.js

```diff
@@ -14,38 +14,89 @@ import domUpdates from './domUpdates';
 
 // ** Button Query Selectors **
 const darkLightModeButton = document.querySelector('.dark-light-mode-js');
+const homeButton = document.querySelector('.go-home-js');
 const currentTripsButton = document.querySelector('.go-to-current-js');
 const plannedTripsButton = document.querySelector('.go-to-planned-js');
 const pastTripsButton = document.querySelector('.go-to-past-js')
-const homeButton = document.querySelector('.go-home-js')
-const nextDestinationButton = document.querySelector('.next-destination-js');
+const makeBookingButton = document.querySelector('.make-booking-js');
 
 // ** Display Query Selectors **
+const homeDisplay = document.querySelector('.dashboard-js');
+const currentDisplay = document.querySelector('.detailed-current-trips-js');
+const plannedDisplay = document.querySelector('.detailed-planned-trips-js');
+const pastDisplay = document.querySelector('.detailed-past-trips-js');
+const bookingDisplay = document.querySelector('.booking-js');
 
-
+// ** Event Listeners **
 darkLightModeButton.addEventListener('click', toggleDarkMode);
+homeButton.addEventListener('click', showHome);
+currentTripsButton.addEventListener('click', showCurrentDetails);
+plannedTripsButton.addEventListener('click', showPlannedDetails);
+pastTripsButton.addEventListener('click', showPastDetails);
+makeBookingButton.addEventListener('click', showBookingDetails);
+
 
-let traveler;
-let travelers = [];
-let trips = [];
-let destinations = [];
+window.traveler;
```

Comment:
I know you said it's not ideal in the PR, but I really love that you poked around, asked around and found a way to make this functional! it's really cool that this ended up working for you.